### ISSUE TITLE
unexpected error after any slcli command

### ISSUE THREAD

**Issue by jjcam** [softlayer/softlayer-python#1045]
whenever running an slcli command, the command completes but is always followed by:

An unexpected error has occured:
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/SoftLayer/CLI/core.py", line 182, in main
    cli.main(**kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Library/Python/2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Python/2.7/site-packages/click/core.py", line 1102, in _process_result
    **ctx.params)
  File "/Library/Python/2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
TypeError: output_diagnostics() got multiple values for keyword argument 'verbose'

Feel free to report this error as it is likely a bug:
    https://github.com/softlayer/softlayer-python/issues

**Comment by ninjabox** [comment]
Wow, that was strange.. I literally just ran into this, tried to fix it real quick, then came here to report it and saw that you had just reported it less than a minute before me.

**Comment by jjcam** [comment]
Glad its not just me

**Comment by acamacho82** [comment]
Could you please provide more information about your environment? 

I tested the slcli on Ubuntu/ElementaryOs and I couldn't reproduce the issue

```
$ python --version
Python 2.7.12
$ slcli --version
slcli (SoftLayer Command-line), version 5.5.2
```

**Comment by jjcam** [comment]
Python 2.7.10
slcli 5.5.2

running on Mac High Sierra 10.13.6

**Comment by tanucojp** [comment]
I've same issue;

$ python --version
Python 2.7.12

$ slcli --version
slcli (SoftLayer Command-line), version 5.5.2

$ slcli loadbal list
:....:.............:..........:.............:....................:......:
: ID : VIP Address : Location : SSL Offload : Connections/second : Type :
:....:.............:..........:.............:....................:......:
:....:.............:..........:.............:....................:......:
An unexpected error has occured:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/SoftLayer-5.5.2-py2.7.egg/SoftLayer/CLI/core.py", line 182, in main
    cli.main(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/Click-7.0-py2.7.egg/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/Click-7.0-py2.7.egg/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/Click-7.0-py2.7.egg/click/core.py", line 1102, in _process_result
    **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/Click-7.0-py2.7.egg/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/Click-7.0-py2.7.egg/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/Click-7.0-py2.7.egg/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
TypeError: output_diagnostics() got multiple values for keyword argument 'verbose'

Feel free to report this error as it is likely a bug:
    https://github.com/softlayer/softlayer-python/issues
The following snippet should be able to reproduce the error

**Comment by acamacho82** [comment]
The issue is because there is an incompatibility with the latest Click library version, and this was reported a few days ago #1040 

In the mean while I can suggest installing the previous version 6.7
```
pip install click==6.7
```

My ubuntu has that version and I also installed it into a windows 10 and it worked

**Comment by jjcam** [comment]
yes - that's worked for me - many thanks

**Comment by tanucojp** [comment]
Hi, it works fine, thank you!

**Comment by allmightyspiff** [comment]
#1055 updates slcli to support click v7, and will be included in v5.6.0
